app/src/keymap.c

```diff
@@ -86,6 +91,14 @@ int zmk_keymap_layer_deactivate(u8_t layer)
 	SET_LAYER_STATE(layer, false);
 };
 
+int zmk_keymap_layer_toggle(u8_t layer)
+{
+	if (zmk_keymap_layer_active(layer))
```

Comment:
Please add `{` brackets for this check. We'll use them even for simple checks like this. Thanks!